Tests/Container/ContainerTest.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Framework\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Zest\Container\Container;
+use Zest\http\Message;
+
+class ContainerTest extends TestCase
+{
+    public function testContainer()
+    {
+        $container = new Container();
+        $msg = new Message();
+
+        $instance = $container->register([Message::class, 'Message'], $msg, false);
+
+        $this->assertFalse($container->isSingleton('Message'));
+
+        $instance2 = $container->register([Message::class, 'Message'], $msg, true);
+
+        $this->assertTrue($container->isSingleton('Message'));
+
+        try {
+            $instance3 = $container->register(['Invalid', 'Invalid'], null, true);
+        } catch (\InvalidArgumentException $e) {
+            $this->assertSame('Claass should be valid instance of Invalid.', $e->getMessage());
```

Comment:
If possible, try to consider using [PHPUnit exceptions](https://phpunit.readthedocs.io/en/8.4/writing-tests-for-phpunit.html#testing-exceptions) test to replace the `try...catch` clause.

Reply:
@peter279k can you take a look again, please ?

Reply:
Hi all, sorry for my delay because of recent busy days.

I will review this PR now.